compiler/hash-attrs/src/ty.rs

```diff
@@ -98,3 +102,82 @@ impl AttrTy {
         self.params.at_index(index).map(|param| param.borrow().ty)
     }
 }
+
+fn repr_ty_from_primitive_ctor(ctor: PrimitiveCtorInfo) -> Option<ReprTyId> {
```

Comment:
These two functions seem a bit out of place here

Reply:
I wanted to put them in `tir-utils` but it causes a dependency cycle, aby suggestions where to put it

Reply:
what is the dependency cycle

Reply:
okay.. still quite annoying, i guess hash-repr has the same issue

Reply:
It is the other way around